src/main/java/de/nordgedanken/auto_hotkey/sdk/AhkSdkType.java

```diff
@@ -0,0 +1,86 @@
+package de.nordgedanken.auto_hotkey.sdk;
+
+import com.google.common.flogger.FluentLogger;
+import com.intellij.openapi.projectRoots.*;
+import de.nordgedanken.auto_hotkey.AHKIcons;
+import org.jdom.Element;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+/**
+ * Controls how the AutoHotkey Sdk type will look and work in the IDE. Registered in plugin.xml
+ */
+public final class AhkSdkType extends SdkType {
+	private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+
+	public AhkSdkType() {
+		super("AutoHotkeySDK");
+	}
+
+	@Override
+	public Icon getIcon() {
+		return AHKIcons.EXE;
+	}
+
+	@Override
+	public @NotNull String suggestHomePath() {
+		return "C:\\Program Files\\AutoHotkey";
+	}
+
+	/**
+	 * Verified that there is an "AutoHotkey.exe" in the directory the user selects
+	 */
+	@Override
+	public boolean isValidSdkHome(String selectedSdkPath) {
+		try (Stream<Path> paths = Files.walk(Paths.get(selectedSdkPath), 1)) {
+			return paths.filter(Files::isRegularFile)
+					.anyMatch(file -> "AutoHotkey.exe".equals(file.getFileName().toString()));
+		} catch (IOException e) {
+			logger.atSevere().withCause(e).log();
+			return false;
+		}
+	}
+
+	@Override
+	public String getInvalidHomeMessage(String path) {
+		return "AutoHotkey.exe could not be found in the selected folder. Please ensure that AutoHotkey.exe is within the folder that you are selecting";
+	}
+
+	@Override
+	public @NotNull String suggestSdkName(@Nullable String currentSdkName, String sdkHome) {
+		return "AutoHotkey";
+	}
+
+	@Override
+	public final String getVersionString(String sdkHome) {
+		return "unknown version";
```

Comment:
Node for the future: Extract it somehow. Maybe using a small ahk script we ship and run.

Reply:
Yes, I was thinking the same thing. I should've added a TODO comment with that. I already wrote the script to do it; it's an easy one-liner that prints the version to the console:
`FileAppend %A_AhkVersion%, *`
Then we'd just need to execute it and read the console and save that into the SdkType when the user creates a new Sdk